README.md

```diff
@@ -18,14 +18,16 @@ yarn add eslint @einride/eslint-plugin --dev
 
 ### Enable all default rules
 
-Add `@einride/default` to the `extends` array in your ESLint configuration file. In `.eslintrc.js`:
+Add `plugin:@einride/default` to the `extends` array in your ESLint configuration file. In `.eslintrc.js`:
 
 ```js
 module.exports = {
   extends: ["plugin:@einride/default"],
 };
 ```
 
+Note that you can omit the `/eslint-plugin` part of the package name, since it will be automatically assumed by ESLint.
```

Comment:
Where is this part?

Reply:
It's a clarification on why you're extending `plugin:@einride/default` rather than `plugin:@einride/eslint-plugin/default`. We could remove it if you prefer not to have it!